### PR TITLE
Added Last Updated Info to Bottom of Course Pages

### PR DESCRIPTION
**Pull request by clintonb**
screen shot 2014-09-19 at 12 43 07 am

@rocha @dsjen

### CODE REVIEW

File: acceptance_tests/__init__.py [hunk 1]
```diff
@@ -141,6 +159,7 @@ def test_page(self):
         """
         self.page.visit()
         self._test_footer()
+        # self._test_data_update_message()
```
Comment (brianhw) [inline acceptance_tests/__init__.py]: should this be commented out?

Comment (clintonb) [inline acceptance_tests/__init__.py]: The tests passed! Fixing.

File: analytics_dashboard/static/js/enrollment-geography-main.js [hunk 2]
```diff
@@ -3,43 +3,32 @@
  * the libraries and kicks off the application.
  */
 
-require(['vendor/domReady!', 'load/init-page'], function(doc, page){
+require(['vendor/domReady!', 'load/init-page'], function (doc, page) {
```
Comment (brianhw) [inline analytics_dashboard/static/js/enrollment-geography-main.js:6]: why the space after function?

Comment (clintonb) [inline analytics_dashboard/static/js/enrollment-geography-main.js:6]: I used PyCharm's file reformat command. The spacing is part of the default JS format.

File: analytics_dashboard/courses/tests/test_presenters.py [hunk 3]
```diff
@@ -48,72 +15,48 @@ def setUp(self):
...
 
-        # check the summary
-        self.assertEqual(summary['interval_end'], datetime.date(year=2014, month=1, day=7))
-        self.assertEqual(summary[AT.ANY], 0)
-        self.assertIsNone(summary[AT.PLAYED_VIDEO])
-        self.assertEqual(summary[AT.ATTEMPTED_PROBLEM], 200)
-        self.assertEqual(summary[AT.POSTED_FORUM], 1234)
-
-    def test_build_summary(self):
-        Switch.objects.create(name='show_engagement_forum_activity', active=False)
```
Comment (dsjen) [inline analytics_dashboard/courses/tests/test_presenters.py:91]: What's the motivation for removing these tests?  Private?

Comment (clintonb) [inline analytics_dashboard/courses/tests/test_presenters.py:91]: They are private, yes; but, the same code is already being tested by `test_get_summary_and_trend_data`.

File: analytics_dashboard/courses/tests/utils.py [hunk 4]
```diff
@@ -2,11 +2,16 @@
 import csv
 import datetime
 
+from analyticsclient.client import Client
 import analyticsclient.constants.activity_type as AT
 
 from courses.permissions import set_user_course_permissions
 
 
+CREATED = datetime.datetime(year=2014, month=2, day=2)
```
Comment (dsjen) [inline analytics_dashboard/courses/tests/utils.py]: `CREATED` is a little generic.  What do you think about `CREATED_DATE`?

Comment (clintonb) [inline analytics_dashboard/courses/tests/utils.py]: Done.

File: analytics_dashboard/courses/tests/test_presenters.py (hunk 3 above)
Comment (dsjen) [inline analytics_dashboard/courses/tests/test_presenters.py:91]: I'm fine not testing private methods.  The one path not covered is when the forum feature is enabled.  The removed tests set `show_engagement_forum_activity` to `True` to make sure that forum data was returned.

Comment (clintonb) [inline analytics_dashboard/courses/tests/test_presenters.py:91]: Updated.

File: analytics_dashboard/courses/tests/test_presenters.py [hunk 5]
```diff
@@ -48,72 +16,55 @@ def setUp(self):
...
-        # pylint: disable=protected-access
-        summary = self.presenter._build_summary(mock_activity_trend_data())
+        if not include_forum_activity:
+            del expected_summary[AT.POSTED_FORUM]
 
-        # make sure that we get the time from "ANY"
-        self.assertEqual(summary['interval_end'], datetime.date(year=2014, month=1, day=7))
+        expected_summary['last_updated'] = CREATED_DATETIME
```
Comment (dsjen) [inline analytics_dashboard/courses/tests/test_presenters.py:62]: Is there a missing assertion for the summary?

**Comment by dsjen** [comment]
Looks good to me!

**Pull request merged**